src/main/java/domain/Bridge.java

```diff
@@ -0,0 +1,36 @@
+package domain;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum Bridge {
+    EXIST(true, "-----"),
+    NON_EXIST(false, "     ");
+
+    private final boolean isExist;
+    private final String format;
+
+    private Bridge(boolean isExist, String format) {
```

Comment:
```suggestion
   Bridge(boolean isExist, String format) {
```

Enum 클래스는 private 키워드를 붙여주지 않아도 되는데요. 이유가 무엇일까요?!

Reply:
해당 내용을 찾아보니 enum은 고정된 상수의 집합이기 때문에 컴파일 시점에 메모리에 모든 enum값들이 메모리에 올라가야 하고, 동적으로 다른 클래스에서 접근해 내부 값을 조작하면 안되기 때문에, Java언어 자체에서 private생성자만 지원하고 있습니다. 그래서 public, protected접근제어자의 경우 컴파일 에러가 나고, private생성자만 가질 수 있기 때문에 private접근제어자는 생략가능하단 사실을 알게됐습니다.

추가적으로 enum에 선언된 상수들은 컴파일 시점에 enum클래스의 각각 static final필드로 초기화되고, Java에서 이를 싱글톤으로 활용하기 위해 기본적으로 private생성자를 이용하고 있고, 이로 인해 enum에서 특정한 상태를 관리하게 될 경우 멀티스레드 환경에서 위험할 수 있다는 사실을 알게됐습니다!